docs/guide-for-local-setup/README.md

```diff
@@ -0,0 +1,40 @@
+# This is a document that describes how to run RDS app locally with local backend
+
+## 1. Setup the Backend
+
+- Go to the RDS Backend setup guide [here](https://github.com/Real-Dev-Squad/website-backend/#readme)
+- Replace the `cors` config in `config/default.js` with the following:
+
+```javascript
+ cors: {
+     allowedOrigins: /(http:\/\/localhost:[0-9]{4}$)/,
+ },
+```
+
```

Comment:
Instead of replacing the default, we should change config/local.js for adding local configs

Reply:
done